code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -306,7 +303,6 @@
 	name = "medical hardsuit"
 	desc = "A special suit that protects against hazardous, low pressure environments. Built with lightweight materials for easier movement."
 	item_state = "medical_hardsuit"
-	slowdown = 1
```

Comment:
Actually, bio armor does nothing--it's literally a worthless stat.

Permeability coefficient is what governs how well viruses and chemicals penetrate a suit, not bio armor.